staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/webhook_converter.go

```diff
@@ -178,6 +180,12 @@ func (c *webhookConverter) Convert(in runtime.Object, toGV schema.GroupVersion)
 		out.SetAPIVersion(toGV.String())
 		return out, nil
 	}
+
+	trace := utiltrace.New(fmt.Sprintf(
+		"Call conversion webhook: custom resource definition: %s, desired API version: %s, object count: %d, UID: %v",
+		c.name, request.Request.DesiredAPIVersion, len(request.Request.Objects), request.Request.UID))
+	defer trace.LogIfLong(5 * time.Second)
```

Comment:
based on len(request.Request.Objects), we know if this is a single resource or within our namespace limits, right?

Reply:
We could select an appropriate boundary from our scale guidelines

Reply:
Thanks, yes, this is a great way to do it. I've gone with: 50ms (per request overhead) + 8 ms * objectCount (per object SLO of conversion webhook + apiserver side serialize/deserialize cost).

Reply:
Fixed.